### PR TITLE
Create butt_sux.py ETESYNC-423

### PR DESCRIPTION
**Pull request by n5d4**
this is the best pr i have ever opened.

### CODE REVIEW

**Review by synft — COMMENTED** [review]
ECOM-487 #comment idk if this will work or not but we shall try - did not work at all. boo

**Comment by synft** [comment]
This is a comment in the conversation tab

**Comment by synft** [comment]
This is a comment in the conversation tab

**Comment by synft** [comment]
This is a comment in the conversation tab

**Comment by synft** [comment]
This is a comment in the conversation tab

**Comment by synft** [comment]
This is a comment in the conversation tab number 6

**Comment by synft** [comment]
This is a comment in the conversation tab number 7

**Comment by synft** [comment]
conv comment send everything

**Review by synft — COMMENTED** [review]
comment to test pr review comments (ba)

**Comment by synft** [comment]
conversation comment - ete repository now

**Review by synft — COMMENTED** [review]
Review comment tester

**Comment by synft** [comment]
commit comment test for github to jira

**Comment by synft** [comment]
conversation comment 228

File: butt_sux.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+print('brett sux butt')
```
Comment (n5d4) [inline butt_sux.py:1]: Review comment1

File: butt_sux.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+print('brett sux butt')
+print('brett sux butt')
+print('brett sux butt')
```
Comment (n5d4) [inline butt_sux.py:4]: Review comment2

**Review by n5d4 — COMMENTED** [review]
Finish your review comment

**Comment by n5d4** [comment]
Test conversation comment. 1:07

**Review by n5d4 — COMMENTED** [review]
this is a review comment

**Comment by n5d4** [comment]
Another one

**Review by synft — APPROVED** [review]
ETESync-423 Nick asked me to approve this

**Comment by n5d4** [comment]
comment

File: butt_sux.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (n5d4) [inline butt_sux.py:3]: nice

**Comment by n5d4** [comment]
test comment two

**Pull request merged**